LibDeflate.lua

```diff
@@ -312,6 +312,143 @@ do
 	end
 end
 
+-- CRC-16/CRC-32 computation
+local band, bnot, xor, lshift, rshift
+
+if bit ~= nil and bit.blshift ~= nil then
+    band = bit.band
+    bnot = bit.bnot
+    xor = bit.bxor
+    lshift = bit.blshift
+    rshift = bit.blogic_rshift
+elseif bit ~= nil and bit.lshift ~= nil then
+    band = bit.band
+    bnot = bit.bnot
+    xor = bit.bxor
+    lshift = bit.lshift
+    rshift = bit.rshift
```

Comment:
I prefer not use define lshift and rshift, because they can be converted to mudulos and divisions and constant table lookup. See "_pow2" table at the top of LibDeflate.lua.